src/components/activity-header/account-owner.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import AccountOwnerIcon from "../../assets/svg-icons/account-circle-icon.svg";
+
+import "./account-owner.scss";
+
+interface IProps {
+  userName: string;
+}
+export class AccountOwnerDiv extends React.PureComponent <IProps> {
+  render() {
+    return (
+      <div className={`accountOwner`} data-cy="account-owner">
```

Comment:
We don't need the extra layer of syntax around the className - we can just use quotes directly and write this as:
`<div className="account-owner" data-cy="account-owner"`